.github/workflows/pr-build.yml

```diff
@@ -61,30 +103,12 @@ jobs:
       - name: validate charts
         run: ./.github/workflows/helm.sh validate
 
-  snyk-scan:
```

Comment:
please add the replacement for this, depedency-check (we should be using both dependency check and trivy) . Example:
https://github.com/hypertrace/java-grpc-utils/pull/46/files#diff-cf34a98fd25d1698b625c9a988868d4cef5036692a76d1e28dc5aa42be237b9eR49-R54